src/shared/components/interactive-tag.tsx

```diff
@@ -0,0 +1,17 @@
+import { TagContent } from "../../features/posts/types/tag-content"
+
+export const TagLabel = ({name, color,id,handleTagClick,tagsId} : TagContent)  => {
```

Comment:
ja bym nie przekazywał handlerów przez componenty, no bo wychodzi na to, że kontener zarządza tym, a to nie jego odpowiedzialność, jak już nazywasz coś kontenerem, to renderuje to cały ui skomponowany z mniejszych komponentów, i ma w sobie jakąś wspoldzieloną logikę, tutaj ewidentnie to jest responsibility interactiveTag zeby zhandlowalTagClick